pennylane/resource/error/error.py

```diff
@@ -142,3 +143,25 @@ def get_error(approximate_op: Operator, exact_op: Operator):
         m1 = qml.matrix(exact_op, wire_order=wire_order)
         m2 = qml.matrix(approximate_op, wire_order=wire_order)
         return qml.math.max(qml.math.svd(m1 - m2, compute_uv=False))
+
+
+def _compute_algo_error(tape) -> Dict[str, AlgorithmicError]:
+    """Given a quantum circuit (tape), this function computes the algorithmic error
+    generated by standard PennyLane operations.
+
+    Args:
+        tape (.QuantumTape): The quantum circuit for which we compute errors
+
+    Returns:
+        dict[str->.AlgorithmicError]: dict with error name and combined error as key-value pair.
+    """
+
+    op_errors = {}
```

Comment:
Minor but you can use more descriptive names instead of providing comments, e.g., `error_name` instead of `error_str` etc, then you can get rid of the comments.